livescore/main.py

```diff
@@ -107,26 +121,52 @@ def getBlueScoreArea(self, img):
         score_area = self.getScoreArea(img)
         return score_area[:, score_area.shape[1]/2:score_area.shape[1]]
 
+    def getRedTeamsArea(self, img):
+        template_width = self.TEMPLATE_RED_TEAMS.shape[1]
+        img_width = img.shape[1]
+        template = imutils.resize(self.TEMPLATE_RED_TEAMS,
+                                  width=int(template_width/1280.0*img_width))
+        top_left, bottom_right = self.matchTemplate(img, template)
+
+        return img[top_left[1]:bottom_right[1], top_left[0]:bottom_right[0]]
+
+    def getBlueTeamsArea(self, img):
+        template_width = self.TEMPLATE_BLUE_TEAMS.shape[1]
+        img_width = img.shape[1]
+        template = imutils.resize(self.TEMPLATE_BLUE_TEAMS,
+                                  width=int(template_width/1280.0*img_width))
+        top_left, bottom_right = self.matchTemplate(img, template)
+
+        return img[top_left[1]:bottom_right[1], top_left[0]:bottom_right[0]]
```

Comment:
As these two methods have almost identical code, it might be nice to factor out the commonalities (see `getScoreArea()` with its children: `getRedScoreArea()` and `getBlueScoreArea()` for reference).